docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,474 @@
+
+```js
+// buildExportTx-cchain-avax.ts
+const privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`;
+```
+
+To run AvalancheJS scripts on Mainnet or Fuji, you can simply replace this with the imported private key:
+
+```js
+// buildExportTx-cchain-avax.ts
+const privKey: string = privateKey;
+```
+
+### Step 1 - Open AvalancheJS
+
+Open your AvalancheJS directory and select the **`examples`** folder to view the source code for the examples scripts.
+
+![Image for post](/img/examples.png)
+
+### Step 2 - Modify Your Export
+
+:::info
+
+The script in this tutorial uses **X Chain <-> C Chain** transfers. Depending on which chains you want to interact with, the scripts you use could be different. Please check the table above for more information.
+
+:::
+
+<br></br>
+
+Select the **`examples/avm`** folder to view the AvalancheJS X-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [`avm/buildExportTx-cchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts)
+
+![Image for post](/img/ajs-export-cchain-res.png)
+
+**Modify Your Avalanche Network Configuration**
+
+Depending on how you started your node, you might use custom settings for your Avalanche Network
+
+_Note: See [Advanced Configurations](https://docs.avax.network/nodes/build/set-up-node-with-installer#advanced-node-configuration) to learn more about custom Node configurations_
+
+Below are the following objects in your Avalanche Network:
+
+```js
+const ip: string = "172.11.111.123";
+const port: number = 9650;
+const protocol: string = "http";
+const networkID: number = 5;
+```
+
+**IP Address**
+
+If you're using a local node:
+
+```js
+const ip: string = "localhost";
+```
+
+Or a custom IP address:
+
+```js
+const ip: string = "<YOUR-NODE-IP-ADDRESS-HERE>";
+```
+
+You can also perform this tutorial on the Avalanche Mainnet or Fuji Testnet by connecting to the [Avalanche Public API](../apis/avalanchego/public-api-server.md) server:
+
+Mainnet:
+
+```js
+const ip: string = "api.avax.network";
+```
+
+Fuji Testnet:
+
+```js
+const ip: string = "api.avax-test.network";
+```
+
+**Port Number**
+
+Default port number:
+
+```js
+const port: number = 9650;
+```
+
+**Protocol**
+
+Local node:
+
+```js
+const protocol: string = "http";
+```
+
+[Avalanche Public API](../apis/avalanchego/public-api-server.md):
+
+```js
+const protocol: string = "https";
+```
+
+**Network ID**
+
+Depending on the networkID which is passed in when instantiating Avalanche, the encoded addresses used will have a distinctive Human Readable Part(HRP) per each network.
+
+_Example Address: 5 - X-`fuji`19rknw8l0grnfunjrzwxlxync6zrlu33yxqzg0h_
+
+Mainnet:
+
+```js
+const networkID: number = 1;
+```
+
+Fuji Testnet:
+
+```js
+const networkID: number = 5;
+```
+
+You can learn more about our encoded addresses [here](cross-chain-transfers.md#network-id)
+
+**Edit the amount of AVAX you want to send:**
```

Comment:
I felt is it very redundant to put every thing again in Fuji flow. Please rewrite it and I will review again.

Reply:
https://github.com/ava-labs/avalanche-docs/pull/782#issuecomment-1142300973

Ok we are moving away from this?

Reply:
Adding my reply from Slack to keep the discussion whole here:

No, I would like the doc to follow that to be clear on each flow, by separating out local and fuji.

Current doc has repeated almost the same in both Local and Fuji which is very redundant and unclear

This is the main point: `As a developer, I would like to follow through this tutorial from start to end to make it work on one environment first, before I try on another.`